Makefile

```diff
@@ -65,5 +67,20 @@ $(OUTPUTDIR)/login.txt:
 	echo "server=$(SERVERADDR)" > $@
 
 .PHONY: clean
+
+clean:
 	rm -rf bin
-	rm $(ASSETDIR)/assets.go
\ No newline at end of file
+	rm -f $(ASSETDIR)/assets.go
+
+.PHONY: clean-assets
+
+clean-assets:
+	rm -f $(ASSETDIR)/assets.go
+
+.PHONY: assets
```

Comment:
`make assets` would only be used when adding sprites or other assets.

it cleans the old assets.go and rebuilds it

actually the assets.go file **only** will rebuild if it doesn't exist, or someone types `make assets`

this is because the actual asset embedding is going on in the `make client/assets/assets.go` target